### PR TITLE
Capture Sentry exceptions when querying from gh graphql api

### PR DESCRIPTION
**Pull request by hubertdeng123**

### CODE REVIEW

File: src/utils/githubEventHelpers.ts [hunk 1]
```diff
@@ -63,7 +73,17 @@ export async function getAllProjectFieldNodeIds(projectFieldId: string, octokit:
     }
   }`;
 
-  const response: any = await octokit.graphql(queryForPrjectFieldNodeIDs);
+  let response: any;
+
+  try {
+    response = await octokit.graphql(queryForPrjectFieldNodeIDs);
```
Comment (chadwhitacre) [inline src/utils/githubEventHelpers.ts:79]:
```suggestion
    response = await octokit.graphql(queryForProjectFieldNodeIDs);
```

Comment (hubertdeng123) [reply src/utils/githubEventHelpers.ts]: good catch

**Review by chadwhitacre — APPROVED** [review]
It's a little WET to have try/catch everywhere, there could be some wrapper function that does the error handling. It'd be a little complicated though since we want to send progressively more info as we have it, so I'm to land as-is.

**Comment by hubertdeng123** [comment]
> It's a little WET to have try/catch everywhere, there could be some wrapper function that does the error handling.

You're right, let me DRY that up

**Comment by chadwhitacre** [comment]
I like it. 😃

**Pull request merged**